deployment/roles/nlx/defaults/main.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+# Kubernetes namespace
+nlx_namespace: nlx
+nlx_instance: proeftuin
```

Comment:
proeftuin?

Reply:
3 guesses where this role was originally created :-)